edx_name_affirmation/api.py

```diff
@@ -194,23 +146,29 @@ def update_verified_name_status(
         * status (Verified Name Status)
         * verification_attempt_id (int)
         * proctored_exam_attempt_id (int)
+        * platform_verification_attempt_id (int)
     """
     filters = {'user': user}
 
-    if verification_attempt_id:
-        if proctored_exam_attempt_id:
-            err_msg = (
-                'Attempted to update the status for a VerifiedName, but two different '
-                'attempt IDs were given. verification_attempt_id={verification_attempt_id}, '
-                'proctored_exam_attempt_id={proctored_exam_attempt_id}'.format(
-                    verification_attempt_id=verification_attempt_id,
-                    proctored_exam_attempt_id=proctored_exam_attempt_id,
-                )
+    if sum(map(bool, [proctored_exam_attempt_id, verification_attempt_id, platform_verification_attempt_id])) > 1:
```

Comment:
Can't we just choose one instead of raising error? Should we favor the platform_verification_attempt_id? I am afraid if we run into this conflict condition, the user will see blocking errors and cannot continue.

Reply:
I think that is a fair point, although it would introduce a change in behavior. I'll ask in stand up!